src/main/java/org/apache/commons/lang3/LocaleUtils.java

```diff
@@ -169,21 +169,21 @@ private static boolean isNumericAreaCode(final String str) {
         if (countryCode == null) {
             return Collections.emptyList();
         }
-        List<Locale> langs = cLanguagesByCountry.get(countryCode);
-        if (langs == null) {
-            langs = new ArrayList<>();
+        List<Locale> langByCountry = cLanguagesByCountry.get(countryCode);
```

Comment:
Sorry, _langs_ was better IMO, mostly because it is _plural_ and is typed as a List. The new name is singular and therefore confusing (to me).